services/QuillLMS/spec/models/user_email_verification_spec.rb

```diff
@@ -61,23 +61,44 @@
     end
   end
 
+  describe '#send_email' do
+    let(:mailer) { double(:mailer, deliver_now!: true) }
+
+    before do
+      allow(UserMailer).to receive(:email_verification_email) { mailer }
+    end
```

Comment:
Do we need this `allow` when we have an `expect` for the same thing in the only spec in the block?

Reply:
Yes - we have to stub the `UserMailer` functionality for the tests (otherwise we get an error about `.deliver_now!` being called on `nil`), but we still want to ensure that it's actually getting called.